Dockerfile.rhel

```diff
@@ -8,7 +8,16 @@ WORKDIR /go/src/github.com/openshift/odo
 RUN make cross
 
 FROM registry.access.redhat.com/ubi7/ubi
-LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts
+
+LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts-container \ 
+    name=openshift/odo-cli-artifacts \ 
+    io.k8s.display-name=atomic-openshift-odo-cli-artifacts-image \
+    maintainer=devtools-deploy@redhat.com \ 
+    summary="This image containing the Linux, Mac and Windows version of odo"
```

Comment:
"This image contains the Linux...."?

Reply:
Thanks, this was carry forward from copy-paste